pages/notifications.php

```diff
@@ -64,11 +64,11 @@
 $allnotifs = has_capability('block/advnotifications:managenotifications', $context);
 $ownnotifs = false;
 
+$bcontext = context_block::instance($blockinstance);
 if (!$allnotifs) {
     if (empty($blockinstance) || !isset($blockinstance) || $blockinstance === -1) {
         throw new moodle_exception('advnotifications_err_nocapability', 'block_advnotifications');
     }
-    $bcontext = context_block::instance($blockinstance);
     $ownnotifs = has_capability('block/advnotifications:manageownnotifications', $bcontext);
 }
```

Comment:
Same as in your other contribution, just checking if this will still work if a user that can't view all notifications are viewing the page and there is no `$blockinstance`?

Reply:
It should not be included in this commit, I'm sorry.
I've fixed it on the other pull request.

Reply:
I reverted this